src/transformers/modeling_tf_utils.py

```diff
@@ -1157,6 +1158,65 @@ def _from_config(cls, config, **kwargs):
         """
         return cls(config, **kwargs)
 
+    def invert_attention_mask(self, encoder_attention_mask: tf.Tensor) -> tf.Tensor:
```

Comment:
Done! I didn't touch the PyTorch side yet because that's a bigger refactor that touches several models, but I can do it in another PR after this if you want.